### PR TITLE
W-053

### PR DESCRIPTION
**Pull request by szymon-muszynski**

### CODE REVIEW

File: backend/plortal/src/main/java/com/plortal/plortal/model/Locations.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package com.plortal.plortal.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
```
Comment (MartynaSzymanskaGitHub) [inline backend/plortal/src/main/java/com/plortal/plortal/model/Locations.java:7]: add 
@NoArgsConstructor
@AllArgsConstructor

File: backend/plortal/src/main/java/com/plortal/plortal/model/Locations.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package com.plortal.plortal.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
```
Comment (MartynaSzymanskaGitHub) [inline backend/plortal/src/main/java/com/plortal/plortal/model/Locations.java:6]: add
@RequiredArgsConstructor

File: backend/plortal/src/main/java/com/plortal/plortal/model/Locations.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+@Getter
+@Setter
+public class Locations {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "locationId")
+    private int id;
+
+    @NonNull
+    @Column(name = "name", length = 100)
```
Comment (MartynaSzymanskaGitHub) [inline backend/plortal/src/main/java/com/plortal/plortal/model/Locations.java:19]: add
@NotBlank

**Review by MartynaSzymanskaGitHub — CHANGES_REQUESTED** [review]
changes described in comments

**Review by MartynaSzymanskaGitHub — APPROVED** [review]
approved

**Review by wojciechwelfle — APPROVED** [review]
looks good

**Pull request merged**